__tests__/integrations/FacebookPixel.test.js

```diff
@@ -0,0 +1,234 @@
+import FacebookPixel from "../../src/integrations/FacebookPixel/browser";
+
+beforeAll(() => { });
+
+afterAll(() => {
+  jest.restoreAllMocks();
+});
+
+const mockAnalyticsIdentifiedUser = {
```

Comment:
```suggestion
const mockedAnalytics= {
```

Reply:
Move this inside specific `test(..)s` keep the same name but make it scoped for each tests

Reply:
updated